2018-03-28/azblob/zt_examples_test.go

```diff
@@ -930,8 +932,8 @@ func ExampleBlobURL_startCopy() {
 	// Create a ContainerURL object to a container where we'll create a blob and its snapshot.
 	// Create a BlockBlobURL object to a blob in the container.
 	u, _ := url.Parse(fmt.Sprintf("https://%s.blob.core.windows.net/mycontainer/CopiedBlob.bin", accountName))
-	blobURL := azblob.NewBlobURL(*u,
-		azblob.NewPipeline(azblob.NewSharedKeyCredential(accountName, accountKey), azblob.PipelineOptions{}))
+	credential, _ := azblob.NewSharedKeyCredential(accountName, accountKey)
+	blobURL := azblob.NewBlobURL(*u, azblob.NewPipeline(credential, azblob.PipelineOptions{}))
```

Comment:
given this is an example (and thus a recommended way of using this library) - shouldn't this be checking that the error object returned is nil?

in addition - if `u` is nil here (as the error isn't caught above) there's a panic here?